libs/ml/include/ml/core/graph.hpp

```diff
@@ -151,6 +152,8 @@ class Graph
 
   std::vector<std::pair<std::string, std::vector<std::string>>> Connections();
 
+  fetch::ml::ops::MLChargeAmount ChargeForwardTo(std::string const &node_name);
```

Comment:
There is no "ForwardTo" method in Graph, I was imagining a naming convention with a ChargeX where X has a one to one correspondence with specific function calls. Is that not possible for some reason?

Reply:
the intention here was to achieve more smooth naming, because this method takes a node name as argument, so to emphasize the fact it is not an estimation for the _whole_ forward pass but only a forward pass "to" this particular node, I named it
... ForwardPassTo(first_convolutional_node)  <-- smth like that.

Reply:
But you're right it is better to keep them consistent